aarhusstadsarkiv/digital-archive
# Get master up to date with development (#70)

* Add caching to CircleCI

* Split check & test jobs

* Fix whitespace in .circleci/config.yml

* Move codecov upload comment

* Iss60 ci build deploy (#63)

* Testing CircleCI tagging

* Test branch & tag

* Add check version shell script to check git tag vs. project version, update circleci config with deploy job

* Fix version

* Change branch filtering for test purposes

* Add in test of git tag vs. version, this should fail horribly

* Add bump.fish for easy bumping of versions. Uses poetry version and updates __init__.py accordingly

* Fix versions

* Change branch filtering back to master

* Update deploy settings so they do not fire at push to master (#65)

* Testing CircleCI tagging

* Test branch & tag

* Add check version shell script to check git tag vs. project version, update circleci config with deploy job

* Fix version

* Change branch filtering for test purposes

* Add in test of git tag vs. version, this should fail horribly

* Add bump.fish for easy bumping of versions. Uses poetry version and updates __init__.py accordingly

* Fix versions

* Change branch filtering back to master

* Ignore all branches for deploy job, I think that's necessary

* Fix devel (#69)

* Release version 0.2.0 (#64)

* Add caching to CircleCI

* Split check & test jobs

* Fix whitespace in .circleci/config.yml

* Move codecov upload comment

* Iss60 ci build deploy (#63)

* Testing CircleCI tagging

* Test branch & tag

* Add check version shell script to check git tag vs. project version, update circleci config with deploy job

* Fix version

* Change branch filtering for test purposes

* Add in test of git tag vs. version, this should fail horribly

* Add bump.fish for easy bumping of versions. Uses poetry version and updates __init__.py accordingly

* Fix versions

* Change branch filtering back to master

* Fix failing deploy job in CircleCI (#67)

* Add caching to CircleCI

* Split check & test jobs

* Fix whitespace in .circleci/config.yml

* Move codecov upload comment

* Iss60 ci build deploy (#63)

* Testing CircleCI tagging

* Test branch & tag

* Add check version shell script to check git tag vs. project version, update circleci config with deploy job

* Fix version

* Change branch filtering for test purposes

* Add in test of git tag vs. version, this should fail horribly

* Add bump.fish for easy bumping of versions. Uses poetry version and updates __init__.py accordingly

* Fix versions

* Change branch filtering back to master

* Update deploy settings so they do not fire at push to master (#65)

* Testing CircleCI tagging

* Test branch & tag

* Add check version shell script to check git tag vs. project version, update circleci config with deploy job

* Fix version

* Change branch filtering for test purposes

* Add in test of git tag vs. version, this should fail horribly

* Add bump.fish for easy bumping of versions. Uses poetry version and updates __init__.py accordingly

* Fix versions

* Change branch filtering back to master

* Ignore all branches for deploy job, I think that's necessary

* Update deploy settings so they do not fire at every master commit

* Fix publish command, it should work now. (#68)

* Add caching to CircleCI

* Split check & test jobs

* Fix whitespace in .circleci/config.yml

* Move codecov upload comment

* Iss60 ci build deploy (#63)

* Testing CircleCI tagging

* Test branch & tag

* Add check version shell script to check git tag vs. project version, update circleci config with deploy job

* Fix version

* Change branch filtering for test purposes

* Add in test of git tag vs. version, this should fail horribly

* Add bump.fish for easy bumping of versions. Uses poetry version and updates __init__.py accordingly

* Fix versions

* Change branch filtering back to master

* Update deploy settings so they do not fire at push to master (#65)

* Testing CircleCI tagging

* Test branch & tag

* Add check version shell script to check git tag vs. project version, update circleci config with deploy job

* Fix version

* Change branch filtering for test purposes

* Add in test of git tag vs. version, this should fail horribly

* Add bump.fish for easy bumping of versions. Uses poetry version and updates __init__.py accordingly

* Fix versions

* Change branch filtering back to master

* Ignore all branches for deploy job, I think that's necessary

* Update deploy settings so they do not fire at every master commit

* Fix publish command, -r wasn't necessary
